mitxpro/settings.py

```diff
@@ -57,6 +57,20 @@
     description="Application-level SSL redirect setting.",
 )
 
+ZENDESK_CONFIG = {
+    "HELP_WIDGET_ENABLED": get_string(
+        "ZENDESK_HELP_WIDGET_ENABLED",
+        False,
+        description="Enabled/disable state for Zendesk web help widget."
+
+    ),
+    "HELP_WIDGET_KEY": get_string(
+        "ZENDESK_HELP_WIDGET_KEY",
+        "8ef9ef96-3317-40a9-8ef6-de0737503caa",
```

Comment:
@pdpinch I believe, We have only one instance for Zendesk yet so I added the key as default one that you had shared with me here but now it will be changeable/configurable for other instances if we will have.